examples/absolute-scale-example/.gitignore

```diff
@@ -0,0 +1,112 @@
+# Logs
```

Comment:
Also would probably merge into the root for maintainability 😌

Reply:
How does that work?
each example is a separate ts project, I don't think I can change the path to tsconfig without modifying esbuild config, or?